### DISCUSSION TITLE
公告 | 关于组织内权限

### DISCUSSION THREAD

**Discussion by WForst-Breeze** [PCL-Community/.github#3] [公告 | Announcements]
组织内成员：

**再次重申** 我们信任每一位组织内成员，因此给予了组织内的所有成员较高的权限。在此处希望各位成员可以合理地进行操作，务必不要出现破坏性操作失误或刻意的破坏性行为。

**建议** 组织内拥有 Owner 权限的组织成员可以合理地进行操作，在操作过程中尽可能保证客观，并妥善保管自身所拥有的权限。

**强调** 组织内的 Owner 权限仅是为了能够更好地服务组织而赋予的。拥有该权限的组织成员不存在与其它成员的所谓 “上下级” 关系。

**呼吁** 组织内成员继续互相监督，如有出现不合理操作的情况出现时，及时进行沟通以最大化减小误会与损失。

<div align="right">

WForst Breeze
2024.06.08

</div>

**Comment by Silverteal** [comment]
Off Topic: 打模联打的

**Comment by wuliaodexiaoluo** [comment]
Off-Topic:Play 舞萌 DX（无恶意）
